src/context/index.tsx

```diff
@@ -14,13 +14,21 @@ interface Props {
 export const AppContextProvider = ({ children }: Props) => {
   const [darkmode, setDarkmode] = useState<boolean>(false);
 
-  const contextValues = { darkmode, setDarkmode };
-  return <AppContext.Provider value={contextValues}>{children}</AppContext.Provider>;
+  const toggleDarkmode = () => {
+    setDarkmode((prevDarkmode) => !prevDarkmode);
+  };
+
+  const contextValues = { darkmode, setDarkmode: toggleDarkmode };
+  return (
+    <AppContext.Provider value={contextValues}>
+      {children}
+    </AppContext.Provider>
+  );
```

Comment:
As mentioned before, make no changes in this file. You have broken some functionality here.

Reply:
I never touched this file what functionality is broken.


```suggestion
  );
```